developers/turret.md

```diff
@@ -0,0 +1,66 @@
+# Turret
```

Comment:
I'm undecided if this should be the new home to the API docs.

Do others have thoughts and opinions?

If we do decide to port the API docs over, I think it would be better if it lived in a subpage.

IE
```
developers/API Docs/endpoint
```

Our YBX docs have an example of how this would be structured https://github.com/script3/yieldblox-docs/tree/master/technical-docs/protocol

Reply:
Also, I think I can just import the generated openApi file into the gitbook ui. This might be an easy add to get all the api docs in for me. I'll investigate.

Reply:
Agree with you about the subpage!

I think I may have misunderstood the purpose of this repo. On the notion brainstorm page, I read "Create a documentation 'hub' for all things Turrets" and thought it was a turrets-documentation-centric thing. Is it intended to be more related to the DocSprint, rather than long-term Turrets documentation?